### PR TITLE
Refactor/typography

### PR DESCRIPTION
**Pull request by ramonxm**
## Description

Atualizando do Typography conforme novo layout.

## Dev Notes

Criado um util para trabalhar com `rem`

## Screenshots / Videos

image

## Developer Checks

- [X] PR title & commits adhere to [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/)
- [X] PR is targeting correct branch, is up-to-date, & no merge conflicts
- [X] Tested on browser device
- [ ] Tested on Mobile device

### CODE REVIEW

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

File: src/utils/convertPxToRem.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const DEFAULT_PIXEL_SIZE = 16;
+
+/** Utility function to convert pixel value to rem
+ * @param {number} numberInPixels - The number in pixels you want to convert to rem.
```
Comment (jonataspinto) [inline src/utils/convertPxToRem.ts:4]: Olha ele, mago do js docs. 

agr falando sério, pq não tipou isso com ts?

**Review by jonataspinto — CHANGES_REQUESTED** [review]

**Review by jonataspinto — APPROVED** [review]

File: src/utils/convertPxToRem.ts (hunk 1 above)
Comment (jonataspinto) [reply src/utils/convertPxToRem.ts:4]: @ramonxm

Comment (ramonxm) [reply src/utils/convertPxToRem.ts:4]: diz o retorno?

File: src/utils/convertPxToRem.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+ * @returns {string} The number in rem
+ */
+const convertPxToRem = (numberInPixels: number): string =>
```
[the remark this answers is not in the document]
Comment (ramonxm) [reply src/utils/convertPxToRem.ts:7]: 😍

**Review by jonataspinto — APPROVED** [review]

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.5 | ✅ |
| First Contentful Paint | 4.3 s |  |
| Time to Interactive | 5.3 s |  |
| Speed Index | 5.9 s |  |
| Total Blocking Time | 430 ms |  |
| Largest Contentful Paint | 6.2 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.96 | ✅ |
| First Contentful Paint | 1.7 s |  |
| Time to Interactive | 4.7 s |  |
| Speed Index | 2.1 s |  |
| Total Blocking Time | 120 ms |  |
| Largest Contentful Paint | 1.7 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

File: src/utils/convertPxToRem.ts (hunk 1 above)
Comment (jonataspinto) [reply src/utils/convertPxToRem.ts:4]: Esquece, viajei!

**Pull request merged**